roles/openshift-networking/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+
```

Comment:
Can we add a simple check to check if the binary exists and cluster is actually up before running these?